tests/test.py

```diff
@@ -4529,310 +4242,61 @@ class TestZoneIDDecryptionWithVaultMasterKeyLoader(HashiCorpVaultMasterKeyLoader
 
 
 class AcraTranslatorTest(AcraTranslatorMixin, BaseTestCase):
+    def get_http_schema(self):
+        return 'https'
 
-    def checkSkip(self):
-        return
+    def get_grpc_channel(self, port):
```

Comment:
why we can't re-use same method from `AcraTranslatorMixin` that configure using same keys?

Reply:
I put it here because here - https://github.com/cossacklabs/acra/blob/master/tests/test.py#L5588 we are using native test call and we should expect here secure grpc channel by default - I think we should delete duplication of this method from `AcraTranslatorMixin`

Reply:
I think, you are right, we can omit that function here